src/infrastructure/dataSchemas/configDataSchema.ts

```diff
@@ -0,0 +1,34 @@
+import { SemanticVersion } from '@/@types/types'
+
+//NOTE: Add any information about StarryDigitizer configurations which is necassary to reconstruct the application states here.
+export interface configDataSchema {
+  schemaVersion: SemanticVersion
+  figure: {
+    scale: number
+  }
+  axisSets: {
```

Comment:
it might be better to use array type annotations here for clarity.

Reply:
fixed in: https://github.com/t29mato/starry-digitizer/pull/183/commits/77ca8f4c652289a6bf16eab0a267f6fa35a4afbb